stable/wavefront/templates/collector-deployment.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if .Values.collector.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name : {{ template "wavefront.fullname" . }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io.instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/component: collector
+  name: {{ template "wavefront.collector.fullname" . }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name : {{ template "wavefront.fullname" .}}
+      app.kubernetes.io/component: collector
```

Comment:
add `app.kubernetes.io/instance`
follow this https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#deployments-statefulsets-daemonsets-selectors

Reply:
I'm not quite clear on what I'm doing incorrectly here.  I do have the component added as a suffix for the name on this deployment.  For this chart that is done as a separate template in _helpers.tpl `wavefront.collector.fullname`. I did similar for the proxy component as well.

https://github.com/puckpuck/charts/blob/wavefront/stable/wavefront/templates/_helpers.tpl#L34-L48

Reply:
@cpanato checking to see if the above explanation makes sense, and if not can you clarify what you need from me to finish this PR.

Thanks.